setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml

```diff
@@ -0,0 +1,51 @@
+# Uses apt-get to install system-wide dependencies and pip to install
+# python dependencies.
+
+# If using Ubuntu 13.04, you may discover that some repositories are missing.
+# See this for details:
+# https://gist.github.com/dergachev/f5da514802fcbbb441a1
+- name: Handle Ubuntu 13.04 end of life.
+  sudo: True
+  shell: sed -i.bak -r 's/(archive|security).ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list
+  when: ansible_distribution == 'Ubuntu' and ansible_distribution_version == '13.04'
+
+- name: Install base Ubuntu packages
+  apt: pkg={{ item }} state=installed update_cache=yes
+  sudo: True
+  with_items:
+    - git
+    - build-essential
+    - python
+    - python-setuptools
+    - python-pip
+    - python-virtualenv
+
+- name: Install python pip
+  sudo: True
+  command: pip install 'pip>=1.5.6' 'setuptools>=5.3'
+
+- name: Install python apt packages
+  apt: pkg={{ item }} state=latest
+  sudo: True
+  with_items:
+    - libicu-dev
+    - libqt4-scripttools
+    - python-enable
+    - python-chaco
+    - python-vtk
+    - python-wxgtk2.8
+    - python-pyside
+    - python-qt4-dev
+    - python-sip
+    - python-qt4-gl
+    - python-software-properties
+
+- name: Update APT repo cache
+  sudo: Yes
+  apt: update_cache=yes
+
+- name: Install python dependencies
+  sudo: True
+  command: >
+             pip install traits traitsui pyserial pylibftdi
+             pyparsing==1.5.7 pygments intelhex six construct sbp==0.29
```

Comment:
What do we think about getting rid of this line and using requirements.txt for the pip installs?
Similarly, can we use pip to install everything except QT?  In this way manual instructions for other os's will be pretty short.  step 1, get python and Qt. Step to, install pikis_tools,